.pre-commit-config.yaml

```diff
@@ -1,14 +1,11 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.1.6
+    rev: v0.2.2
     hooks:
       - id: ruff
-        args: [--fix, --exit-non-zero-on-fix]
-
-  - repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 23.11.0
-    hooks:
-      - id: black
+        args: ["--fix"]
```

Comment:
Why did you remove `--exit-non-zero-on-fix` here? :)

Reply:
Should we have `fix` and `exit-non-zero-on-fix` in `pyproject.toml`, or are they (or just one) better here?

Reply:
I usually put `--fix` in pyproject.toml, so that it also autofixes if you execute ruff directly.

I don't really know much about the pros and cons of `--exit-non-zero-on-fix`, and whether it should be put in `pyproject.toml` or `.pre-commit-config.yaml`!

Reply:
OK, let's just put `--fix` in `pyproject.toml` and leave `--exit-non-zero-on-fix` here :)

Reply:
Will do...